src/components/ApplicationForm/ReviewCards.js

```diff
@@ -88,28 +88,44 @@ const ethnicityOptions = {
   preferNot: 'Prefer not to say',
 }
 
+const eventOptions = {
+  option1: 'Local Hack Day / Hack Camp',
```

Comment:
ONE WORD VINCENT ONE WORD stylization wall of shame u go

Reply:
oof i will fix thank you! i copied that from somewhere else smh